src/components/button/button-link/button-link.ts

```diff
@@ -0,0 +1,81 @@
+import { spread } from '@open-wc/lit-helpers';
+import { nothing, type TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { html, unsafeStatic } from 'lit/static-html.js';
+
+import type { LinkRenderVariables } from '../../core/common-behaviors';
+import {
+  LanguageController,
+  SbbDisabledMixin,
+  SbbIconNameMixin,
+  SbbNegativeMixin,
+  SbbLinkBaseElement,
+  resolveLinkOrStaticRenderVariables,
+  targetsNewWindow,
+} from '../../core/common-behaviors';
+import { isValidAttribute, setAttributes } from '../../core/dom';
+import { i18nTargetOpensInNewWindow } from '../../core/i18n';
+import { SbbButtonCommonElementMixin } from '../common/button-common';
+
+/**
+ * It displays a button enhanced with the SBB Design, which will behave as a link.
+ *
+ * @slot - Use the unnamed slot to add content to the button-link.
+ * @slot icon - Slot used to display the icon, if one is set
+ */
+@customElement('sbb-button-link')
+export class SbbButtonLinkElement extends SbbButtonCommonElementMixin(
+  SbbNegativeMixin(SbbDisabledMixin(SbbIconNameMixin(SbbLinkBaseElement))),
+) {
+  private _language = new LanguageController(this);
+
+  public override connectedCallback(): void {
+    super.connectedCallback();
+    // Check if the current element is nested in an action element.
+    const formField = this.closest?.('sbb-form-field') ?? this.closest?.('[data-form-field]');
+    if (formField) {
+      this.negative = isValidAttribute(formField, 'negative');
+    }
+  }
+
+  protected override render(): TemplateResult {
+    const {
+      tagName: TAG_NAME,
+      attributes,
+      hostAttributes,
+    }: LinkRenderVariables = resolveLinkOrStaticRenderVariables(this);
+
+    // ## Migr: Host attributes ##
+    setAttributes(this, hostAttributes);
+    // ####
+
+    /* eslint-disable lit/binding-positions */
+    return html`
+      <${unsafeStatic(TAG_NAME)} class="sbb-button" ${spread(attributes)}>
```

Comment:
Hmm, is the example missing (after e.g.)?

Reply:
Yes, it was supposed to be `<sbb-button-static>` 😅

Reply:
@kyubisation oh ok, i thought it was more :D

anyway, i have a question:
right now, the `isStatic` flag is set to true if the consumer sets it, or if the consumer uses the sbb-(button/link-button) within a button, or a sbb-(link/button-link) within an anchor;
if i create the new `sbb-button-static` component, does this behavior disappear? consumers will use the static version if needed (the old `isStatic` flag set to true), but what would happen with nested links/buttons?

Reply:
This would be a breaking change which we accept, so no automatic change to static anymore. we just need to check our own usages (e.g. maybe teaser-hero, or more?)

Reply:
Great 👍 

the static case is used also in the breadcrumb, teaser and teaser-hero;
while for the breadcrumb I feel we can remove it without pain (since it's mostly projected in the breadcrumb-group, which would hardly be nested into a link), I'm not sure about the other two, because i don't know exactly their use case.. 
should we create a static component for them, or can I remove the isStatic case from them too?

Reply:
> @kyubisation oh ok, i thought it was more :D

My intention was also for sbb-link-static, but other than that, I don't think we need more.
@jeripeierSBB Anything else that comes to mind?

Reply:
For teaser-hero and teaser we don't need the static case, I did also check if there is an usage somewhere in SBB, but there is no usage of the static case for those components.
In the shadow DOM of the teaser hero we in Lyne are using the sbb-link. this specific link has to be a sbb-link-static, because before it was auto detected as static.
It's a little bit difficult to check whether we have another case about static auto detection. So far I don't remember another case.

Reply:
Static components created in [fbdfd5b](https://github.com/lyne-design-system/lyne-components/pull/2364/commits/fbdfd5b6821a2323fc942b6aa8ce6538fec1d35a) and mixed static/link behavior fixed in [0edc2f6](https://github.com/lyne-design-system/lyne-components/pull/2364/commits/0edc2f6ec801979752c9132c28709fd89f6c86f3)